### DISCUSSION TITLE
Acceso al terminar el curso Full Stack

### DISCUSSION THREAD

**Discussion by Dariopg** [Launch-X-Latam/MisionFrontEnd#403] [General]
Buenas noches, mi duda es si al terminar todo el curso el material de estudio como los videos y los repositorios de github van a seguir estando disponibles, o directamente se nos va a cortar el acceso.
Muchas gracias

**Comment by TonatiuhDSJ** [comment]
Que yo sepa, tienes acceso para siempre, a solo que innovaccion desaparezca, fuera de ello tienes todo el material para consultarlo toda la vida.

**Reply by Dariopg** [reply]
Ha perfecto, muchas gracias.
